src/software/firmware/test/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.5)
+
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
I cannot find reliable information, but it seems arduino sdk is stuck at c++11, for now. To be confirmed.

Reply:
Ok changed to 11